examples/headless.rs

```diff
@@ -356,26 +368,20 @@ mod frame_capture {
 fn setup_gaussian_cloud(
     mut commands: Commands,
     asset_server: Res<AssetServer>,
+    gaussian_splatting_viewer: Res<GaussianSplattingViewer>,
     mut gaussian_assets: ResMut<Assets<GaussianCloud>>,
     mut scene_controller: ResMut<frame_capture::scene::SceneController>,
     mut images: ResMut<Assets<Image>>,
     render_device: Res<RenderDevice>,
 ) {
     let cloud: Handle<GaussianCloud>;
 
-    // TODO: add proper GaussianSplattingViewer argument parsing
-    let file_arg = get_arg(1);
-    if let Some(n) = file_arg.clone().and_then(|s| s.parse::<usize>().ok()) {
-        println!("generating {} gaussians", n);
-        cloud = gaussian_assets.add(random_gaussians(n));
-    } else if let Some(filename) = file_arg {
-        if filename == "--help" {
-            println!("usage: cargo run -- [filename | n]");
-            return;
-        }
-
-        println!("loading {}", filename);
-        cloud = asset_server.load(filename.to_string());
+    if gaussian_splatting_viewer.gaussian_count > 0 {
+        println!("generating {} gaussians", gaussian_splatting_viewer.gaussian_count);
+        cloud = gaussian_assets.add(random_gaussians(gaussian_splatting_viewer.gaussian_count));
+    } else if !gaussian_splatting_viewer.input_file.is_empty() {
+        println!("loading {}", gaussian_splatting_viewer.input_file);
+        cloud = asset_server.load(&gaussian_splatting_viewer.input_file);
```

Comment:
- should we use the `log` function here? & is the `viewer` example meant to also support wasm?

Reply:
yes, I think log would go better here for the web output. `viewer` is meant to support native and web, web viewer is hosted here: https://mosure.github.io/bevy_gaussian_splatting/index.html